sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisTest.scala

```diff
@@ -177,14 +177,18 @@ trait AnalysisTest extends PlanTest {
       caseSensitive: Boolean = true,
       line: Int = -1,
       pos: Int = -1): Unit = {
+
+    def sameMapContents(left: Map[String, String], right: Map[String, String]): Boolean =
```

Comment:
cc @MaxGekk fix the problem I mentioned in https://github.com/apache/spark/pull/38394#discussion_r1005508300